.github/workflows/build.yml

```diff
@@ -164,10 +164,10 @@ jobs:
           - false
           - true
         include:
-          - os: windows-latest # FIXME(diegorusso): change to os: windows-aarch64
+          - os: windows-aarch64
```

Comment:
Is this runner only available for select projects right now?

Do you know when it might be more widely available, and can we add some default for forks in the meantime?

It looks like we're missing Windows entirely on forks:

<table>
<tr>
<th><a href="https://github.com/python/cpython/actions/runs/14111802612?pr=131559">upstream</a>
<th><a href="https://github.com/hugovk/cpython/actions/runs/14112081232">fork</a>
<tr>
<td valign=top>
<img width="321" alt="image" src="https://github.com/user-attachments/assets/bd355fcb-1fef-4df0-b565-3b93e9f9fb7a" />
<td valign=top>
<img width="312" alt="image" src="https://github.com/user-attachments/assets/5fd1ae2c-60b3-40f5-a027-f8fb144ff5cd" />
</table>

Reply:
Good point. At the moment these are available on selected projects for now. I'm not sure when they will be in GA. I will ask and report back.
I'll add something to skip the aarch64 on forks.

Reply:
Can you double check if you have Windows runners available on forks without my patch? See here: https://github.com/diegorusso/cpython/actions/runs/14119011738

Reply:
Also CIFuzz are skipped.

Reply:
Yes, I do have Windows runners on forks without this patch, here's `main` and a branch from `main`:
* https://github.com/hugovk/cpython/actions/runs/14126514909
* https://github.com/hugovk/cpython/actions/runs/14126621995

Note https://github.com/python/cpython/pull/131702 was merged 3 days ago to fix a Windows skip when running on `main`, and I don't think we have that here yet? Let's first update this PR from `main` and re-check.

Reply:
Ah maybe I need to rebase the branch then.

Reply:
Click the magic button:

<img width="177" alt="image" src="https://github.com/user-attachments/assets/89b62fac-8d27-4bb3-81d9-48a6da4a7cee" />

Reply:
Have a look now.